docs/variants/novacustom_ns5x_adl/hardware-matrix.md

```diff
@@ -0,0 +1,59 @@
+# Hardware configuration matrix
```

Comment:
I don't like format of this page, I think we should have something like - at some point I suggested SVG in format used by Phoronix and replacing key elements.